internal/manifests/targetallocator/container.go

```diff
@@ -110,6 +111,24 @@ func Container(cfg config.Config, logger logr.Logger, instance v1alpha1.TargetAl
 		},
 	}
 
+	if cfg.CertManagerAvailability() == certmanager.Available {
+		ports = append(ports, corev1.ContainerPort{
+			Name:          "https",
+			ContainerPort: 8443,
+			Protocol:      corev1.ProtocolTCP,
+		})
+		volumeMounts = append(volumeMounts, corev1.VolumeMount{
+			Name:      naming.TAServerCertificate(instance.Name),
+			MountPath: "/tls",
+		})
+		args = append(args,
+			"--enable-https-server",
+			"--https-ca-file=/tls/ca.crt",
```

Comment:
We use these paths in multiple places, we should turn them into constants. Both the directory (needed for volume mounting), and the filenames themselves (needed for configuration).

Reply:
Would defaultHttpsCaFileName, defaultTLSDirPath, etc. in https://github.com/open-telemetry/opentelemetry-operator/blob/main/internal/config/main.go be the right place?

Reply:
I wouldn't make them configurable at the operator level. Since they're used in TA and Collector manifests both, `manifestutils` would be a good location.